src/components/modal/ModalHeader.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Svg } from '../svg/Svg';
+
+export interface IModalHeaderOwnProps {
+  id?: string;
+  title: string;
+  classes?: string[];
+}
+
+export interface IModalHeaderStateProps { }
+
+export interface IModalHeaderDispatchProps {
+  onClose?: () => void;
+}
+
+export interface IModalHeaderProps extends IModalHeaderOwnProps, IModalHeaderStateProps, IModalHeaderDispatchProps { }
+
+export class ModalHeader extends React.Component<IModalHeaderProps, any> {
+
+  close() {
+    this.props.onClose();
```

Comment:
since it's optional I would add a check `if (this.props.onClose) { ...`

If you only use it internally make it private